index-alphabetic-browse.sh

```diff
@@ -31,11 +31,18 @@ then
   SOLR_HOME="$VUFIND_HOME/solr/vufind"
 fi
 
+# This would be /opt/bitnami/solr on a solr docker container
+if [ -z "$SOLR_JAR_PATH" ]
+then
+  SOLR_JAR_PATH="${SOLR_HOME}/../vendor"
+fi
+
 set -e
 set -x
 
 cd "`dirname $0`/import"
-CLASSPATH="browse-indexing.jar:${VUFIND_HOME}/import/lib/*:${SOLR_HOME}/jars/*:${SOLR_HOME}/../vendor/modules/analysis-extras/lib/*:${SOLR_HOME}/../vendor/server/solr-webapp/webapp/WEB-INF/lib/*"
+LOG4J_CLASSPATH=$(find "${SOLR_JAR_PATH}/server/lib/ext" -name 'log4j*.jar' -printf '%p:' | sed 's/:$//')
+CLASSPATH="browse-indexing.jar:$(echo solrmarc_core*.jar):${VUFIND_HOME}/import/lib/*:${SOLR_HOME}/jars/*:${SOLR_JAR_PATH}/modules/analysis-extras/lib/*:${SOLR_JAR_PATH}/server/solr-webapp/webapp/WEB-INF/lib/*:${LOG4J_CLASSPATH}"
```

Comment:
Is `$(echo solrmarc_core*.jar)` going to cause problems if there are multiple files matching the wildcard pattern in the directory? Obviously, this SHOULDN'T ever happen, but sometimes if people upgrade without following all the recommended best practices, they can end up with multiple copies of SolrMarc, so it's not impossible. (Maybe we don't care, because when this happens, it will also cause other problems and needs to be fixed... but just wondering if we should consider making it more robust).

Reply:
Yes, I was going to do that but I got distracted by pressing bugs. I'll see you at WOLFCON.

Reply:
Thanks! I probably won't have a chance to review and merge this until after the conference, but I'm glad to hear you'll be there -- I look forward to meeting you in person!